src/tasks/predict_modality/methods/guanlab_dengkw_pm/config.vsh.yaml

```diff
@@ -0,0 +1,49 @@
+# The API specifies which type of component this is.
+__merge__: ../../api/comp_method.yaml
+
+functionality:
+
+  name: guanlab_dengkw_pm
+
+  info:
+    label: Guanlab-dengkw
+    summary: A kernel ridge regression method with RBF kernel. 
+    description: |
+      This is a solution developed by Team Guanlab - dengkw in the Neurips 2021 competiton to predeict one modality from another using kernel ridge regression (KRR) with RBF kernel. Truncated SVD is applied on the combined training and test data from modality 1 followed by row-wise z-score normalization on the reduced matrix. The truncated SVD of modality 2 is predicted by training a KRR model on the normalized training matrix of modality 1. Predictions on the normalized test matrix are then re-mapped to the modality 2 feature space via the right singular vectors. 
+    preferred_normalization: log_cp10k
+    reference: lance2022multimodal
+    documentation_url: https://github.com/openproblems-bio/neurips2021_multimodal_topmethods/tree/main/src/predict_modality/methods/Guanlab-dengkw
+    repository_url: https://github.com/openproblems-bio/neurips2021_multimodal_topmethods/tree/main/src/predict_modality/methods/Guanlab-dengkw
+
+  # Component-specific parameters (optional)
+  arguments:
+    - name: "--distance_method"
+      type: "string"
+      default: "minkowski"
+      description: The distance metric to use. Possible values include `euclidean` and `minkowski`.
+      choices: [euclidean, minkowski]
+    - name: "--n_pcs"
+      type: "integer"
+      default: 50
+      description: Number of components to use for dimensionality reduction.
+
+  resources:
+    - type: python_script
+      path: script.py
+
+platforms:
+  - type: docker
+    image: ghcr.io/openproblems-bio/base_python:1.0.2
+    setup:
+      - type: python
+        packages: 
+          - scikit-learn
+          - pandas
+          - numpy
+          - scanpy
+
+  - type: native
+
+  - type: nextflow
+    directives:
+      label: [ "hightime", highmem, highcpu]
```

Comment:
Would the right time label be "hightime" or "longtime"? Seems like `check_method_config.py` and `check_metric_config.py` define "longtime" as one of the valid time labels but in the configs, definitions of time labels for directives use "hightime".

Reply:
Should be `hightime` and not `longtime`! Thanks for noticing this.